cmd/run.go

```diff
@@ -32,13 +29,14 @@ When everything is battoned down, it is time to run forth.`,
 	},
 }
 
+// Re-export exit codes from SDK for backwards compatibility
 const (
-	TestPass = iota
-	TestFail
-	Aborted
-	InternalError
-	BadUsage
-	NoTests
+	TestPass      = command.TestPass
+	TestFail      = command.TestFail
+	Aborted       = command.Aborted
+	InternalError = command.InternalError
+	BadUsage      = command.BadUsage
+	NoTests       = command.NoTests
```

Comment:
```suggestion
```

It seems like `Aborted` is the only one of these that's used, am I reading that right? Or are these referenced by other files?

If not, we should remove these and just directly use `command.Aborted` where it's needed below.